ARAnalyticalProvider.m

```diff
@@ -22,6 +22,12 @@ - (id)initWithIdentifier:(NSString *)identifier {
     return [super init];
 }
 
+- (void)identifyUserWithID:(NSString *)userID anonymousID:(NSString *)anonymousID andEmailAddress:(NSString *)email {
```

Comment:
initially, I was like, nah. 
However, now I'm more like, OK, this could be valid for every use provider.